db/schema.rb

```diff
@@ -840,6 +850,7 @@
     t.string "family_name", null: false
     t.string "session_token"
     t.integer "fallback_role", default: 0, null: false
+    t.string "reporting_app_session_token"
```

Comment:
I don't think you mean to add this column in this PR.

Reply:
ahh, yes, a hangover from the other migration on the spike branch which i had forgotten to rollback before switching. Fixed.